Directory.Build.targets

```diff
@@ -0,0 +1,10 @@
+<Project>
+  <PropertyGroup>
+    <DuendePackageDir Condition=" '$(DuendePackageDir)' == '' ">$(MSBuildThisFileDirectory)packages</DuendePackageDir>
+  </PropertyGroup>
+
+  <Target Name="PackAndMove" Condition="'$(IsPackable)' == 'true'">
+    <Exec Command="dotnet pack --configuration Release --output $(DuendePackageDir)" />
+  </Target>
+
+</Project>
```

Comment:
`dotnet pack -o <OutputDir>`, `dotnet pack -o $env:PACKAGES_PATH` at solution level should be sufficient I'd say so don't think we need a custom build target.